katsdpsigproc/percentile.py

```diff
@@ -72,17 +83,23 @@ def generate(size, wgsy):
             inp.set(queue, host_data)
             return tune.make_measure(queue, fn)
 
-        return tune.autotune(generate,
-                             size=[8, 16, 32, 64, 128, 256, 512, 1024],
-                             wgsy=[1, 2, 4, 8, 16, 32])
+        return cast(_TuningDict, tune.autotune(generate,
```

Comment:
Didn't think I'd see `cast` in Python, but hey :-)

Reply:
Yep :-) It has no runtime effect though - it's just a hint to mypy.